lua/entities/pewpew_base_cannon/init.lua

```diff
@@ -70,7 +70,11 @@ function ENT:SetOptions( BULLET, ply, firekey, reloadkey, FireDirection )
 	if (!self.Ammo) then
 		self.Ammo = self.Bullet.Ammo
 	end
-	
+
+	if (!self.ShotCount) then
+		self.ShotCount = self.Bullet.ShotCount or 1
+	end
```

Comment:
Could this be:
```lua
    self.ShotCount = self.ShotCount or self.Bullet.ShotCount or 1
```

Reply:
probably

Reply:
wait, no

Reply:
it checks if it is there or set and then sets it, so there's no need

Reply:
Right, but what I suggested should do the same thing but in one line, without sacrificing readability

Reply:
oh i see what you mean

Reply:
done